tools/svdconv/SVDConv/src/SVDConv_Msgs.cpp

```diff
@@ -204,10 +204,11 @@ const MsgTable SvdConv::msgTable = {
   { "M384", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "Number of PMU Event Counters (found val: '%NUM%') not set or outside range [2..31]. Ignoring PMU entry."} },   // 14.02.2020
   { "M385", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "PMU not supported for CPU '%NAME%'"                                         } },   // 14.02.2020
   { "M386", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "Name not C compliant: '%NAME%' : Brackets [] found"                         } },
-  { "M387", { MsgLevel::LEVEL_ERROR,    CRLF_B,   ""                                                                           } },
+  { "M387", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "CPU: <sauNumRegions> set to null but regions are configured."               } },
   { "M388", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "MVE Floating-Point support is set but MVE is not set"                       } },
   { "M389", { MsgLevel::LEVEL_ERROR,    CRLF_B,   "Specified <deviceNumInterrupts>: '%NUM%' greater or equal '%NAME%': '%NUM2%'."} },   // 05.05.2020
   { "M390", { MsgLevel::LEVEL_WARNING3, CRLF_B,   "Checking IRQ '%NAME%': CPU unknown (see <cpu>). Assuming a maximum of %NUM% external Interrupts."} },    // 13.05.2020
+  { "M391", { MsgLevel::LEVEL_WARNING,  CRLF_B,   "CPU: configured num of SAU regions '%NUM%' greater than <sauNumRegions> value: %NUM2%"        } },
```

Comment:
configured number of SAU regions ...

or 

<sauNumRegions>

Reply:
fixed